common/walker/src/report.rs

```diff
@@ -123,9 +126,97 @@ impl SplitScannerError for Result<Report, ScannerError> {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, serde::Deserialize, serde::Serialize)]
```

Comment:
I don't see the benefit of this enum. It looks like it only carries a `String`, gets filled with a string and then gets turned back into a string. If that's the case, then let's just use a `String` (or `&str` or whatever works).